zip-0227.rst

```diff
@@ -189,19 +199,20 @@ An issuance action, `IssueAction`, is the instance of issuing a specific custom
 - :math:`\mathsf{asset\_desc}`: the Asset description, a byte string of up to 512 bytes as defined in the `Specification: Asset Identifier`_ section.
-- `notes`: an array containing the unencrypted output notes of the recipients of the Asset, of type `Note`
+- `notes`: an array of `Note` containing the unencrypted output notes of the recipients of the Asset.
 - ``finalize``: a boolean that defines whether the issuance of that specific custom Asset is finalized or not
 
 An asset's :math:`\mathsf{AssetId}` is added to the :math:`\mathsf{previously\_finalized}` set after a block that contains any issuance transaction for that asset with ``finalize = 1``. It then cannot be removed from this set. For Assets with :math:`\mathsf{AssetId} \in \mathsf{previously\_finalized}`, no further tokens can be issued, so as seen below, the validators will reject the transaction. For Assets with :math:`\mathsf{AssetId} \not\in \mathsf{previously\_finalized}`, new issuance actions can be issued in future transactions. These must use the same Asset description, :math:`\mathsf{asset\_desc}`, and can either maintain ``finalize = 0`` or change it to ``finalize = 1``, denoting that this custom Asset cannot be issued after the containing block.
   
 ================= =============================== ========================== ===========================================================================================
-Size              Name                            Data Type                  Description
+Bytes              Name                            Data Type                  Description
 ================= =============================== ========================== ===========================================================================================
-2 bytes           :math:`\mathsf{assetDescSize}`  byte                       The length of the :math:`\mathsf{asset\_desc}` string in bytes
-Varies            :math:`\mathsf{asset\_desc}`    byte                       UTF-8 encoded string, of size :math:`\mathsf{assetDescSize}` bytes
+2           assetDescSize  byte                       The length of the asset\_desc string in bytes
+Varies      asset\_desc    byte                       UTF-8 encoded string, of size assetDescSize bytes
 Varies            nNotes                          compactSize                The number of notes in the issuance action
+TBD         noteSize       byte                   The size, in bytes, of a Note
 noteSize * nNotes vNotes                          Note[nNotes]               A sequence of note descriptions within the issuance action
-1 byte            ``flagsIssuance``               byte                       An 8-bit value with the ``finalize`` boolean value as the LSB, and the other bits set to 0.
+1            ``flagsIssuance``               byte                       An 8-bit value with the ``finalize`` boolean value as the LSB, and the other bits set to 0.
```

Comment:
`flagsIssuance` is used here but formally introduced in `T.5a.iii: flagsIssuance`, consider using a internal link to the relevant section

Reply:
Fixed in https://github.com/QED-it/zips/pull/20/commits/da051d1247f9b7046f21cbb852808e9684089d8a